src/BloomExe/Edit/EditingView.cs

```diff
@@ -421,11 +405,6 @@ public void AddMessageEventListener(string eventName, Action<string> action)
 			_browser1.AddMessageEventListener(eventName, action);
 		}
 
-		public void UpdateTemplateList()
```

Comment:
normally yes, but the templatepagesview was visible while the user was changing stuff, and (I don't know) might have stayed around while the book and collection changed. now we rebuild the (modal) dialog whenever the user clicks the Add Page button, so I don't think it's necessary.